primes.rb

```diff
@@ -1,6 +1,9 @@
+
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+  def self.sum_to(limit)
+  	require 'prime'
+  	Prime.each(limit).inject(:+)
```

Comment:
This will work! I'd encourage you to have a go at writing your own implementation; it's tricky but it makes you think...! https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes